install.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+# Authors:
+#   Sreeja Kamishetty <sreeja25kamishetty@gmail.com>
+
+# Description:
+#   A post-installation bash script for engelsystem
+
+echo "Update your package manager. $ sudo apt-get update."
+apt-get update
+
+echo "Installing LAMP"
+echo "Install Apache"
+apt-get install apache2
+
+echo "Install MySQL"
+apt-get install mysql-server php5-mysql
+
+echo "Install PHP"
+apt-get install php5 libapache2-mod-php5 php5-mcrypt
+
+echo "Install php cgi"
+apt-cache search php5-cgi
```

Comment:
There are some methods to run a PHP website:

  1. Run with PHP embedded in Apache. Apache is the OS process which runs your web PHP scripts.
  2. CGI/FCGI: Run PHP in a separate OS process, this one will run your web PHP scripts. Then you need a separate web server (Apache/Nginx/lighttpd) to talk with this PHP process via some protocol (CGI/FastCGI). Note that Apache in this case doesn't run the same as the case 1.

Based on your setup, installing _libapache2-mod-php5_ and modify _DocumentRoot_ in Apache config, I believe you are using the first method. So no need to install _php5-cgi_.

@DishantK1807 I think this should be shared to you as well.